tests/test_csc.py

```diff
@@ -216,20 +221,24 @@ async def testCantSetPowerWithBulbOff(self):
                 b".0111rTECDrLvb7\r": b"#01110rTECDrLv+0066D3\r",
                 b".0113rTECB2Cr6a\r": b"#01130rTECB2Cr047,C99\r",
             }
-            harness.csc.kiloarcModel.warmupPeriod = 15
-            await harness.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
-            await harness.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
+            self.csc.kiloarcModel.warmupPeriod = 15
+            await self.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
             await asyncio.sleep(3)  # Sleep while we wait for chiller to start chilling.
```

Comment:
Is there something you can await, such as an event, or even a task that is in the CSC, that will let you _know_ that the CSC has started cooling? If so, I strongly suggest you await that instead of `sleep(3)`. It avoids a race condition and makes the test more robust and efficient.

Similarly I suggest looking for `await asyncio.sleep` elsewhere in this file and try to eliminate it. Sometimes it's the only way, but here I am pretty sure there is a better alternative (and I hope it's an event).